### PR TITLE
Universal spacing

### PR DESCRIPTION
**Pull request by Heydon**
```
## 28.0.0 (2022-09-14)
    * BREAKING:
        * New universal block spacing (inter-element margin) for flow content
    * block-spacing alias design tokens
    * spacing applied to heading utility classes
    * “accessible focus styles” removed (redundant `outline: none` on non-interactive elements)
```

## Comparison of brands
Screenshot of brand flow content spacing
Screenshot of brand flow content spacing
Screenshot of brand flow content spacing
Screenshot of brand flow content spacing

### CODE REVIEW

File: context/design-tokens/alias/default/typography/typography.json [hunk 1]
```diff
@@ -81,6 +81,20 @@
 					"value": "{font-weight.bold.value}"
 				}
 			}
+		},
```
Comment (sturobson) [inline context/design-tokens/alias/default/typography/typography.json:84]: 😄

Comment (Heydon) [reply context/design-tokens/alias/default/typography/typography.json:84]: @sturobson  I don't get it 😅 did I put them in the wrong place?

Comment (sturobson) [reply context/design-tokens/alias/default/typography/typography.json:84]: sorry, this was just a "yay, more tokens!" smile 😄

**Review by sturobson — APPROVED** [review]

**Review by foxintherain — APPROVED** [review]

**Comment by sturobson** [comment]
With #878 - I think we can close this @benjclark @alexkilgour ?

**Comment by benjclark** [comment]
@sturobson yeah this can be killed

**Pull request closed without merging**